app/iht/utils/pdf/XmlFoToPDF.scala

```diff
@@ -120,9 +121,10 @@ trait XmlFoToPDF {
     val dateOfPredeceased = applicationDetails.widowCheck.flatMap { x => x.dateOfPreDeceased }
 
     setupCommonTransformerParametersPreAndPost(transformer, registrationDetails, preDeceasedName, dateOfMarriage,
-      applicationDetails.totalAssetsValue, applicationDetails.totalLiabilitiesValue,
-      applicationDetails.totalExemptionsValue, applicationDetails.totalGiftsValue)
+      applicationDetails.totalAssetsValue, applicationDetails.totalLiabilitiesValue, applicationDetails.totalGiftsValue)
 
+    transformer.setParameter("giftsTotalExclExemptions", getOrMinus1(applicationDetails.totalPastYearsGiftsValueExcludingExemptionsOption))
+    transformer.setParameter("exemptionsTotal", getOrMinus1(applicationDetails.totalPastYearsGiftsExemptionsOption))
```

Comment:
This used for the Exemptions Value as shown on the estate overview. Not Exemptions for Gifts